bookkeeper-server/conf/bk_server.conf

```diff
@@ -48,6 +48,12 @@ journalDirectory=/tmp/bk-txn
 # Directories to store index files. If not specified, will use ledgerDirectories to store.
 # indexDirectories=/tmp/bk-data
 
+# Directory where LocalBookKeeper serializes the bookies config objects, which can be 
+# later used by BookieShell for running against LocalBookKeeper.
+# This config value should match DEFAULT_LOCALBOOKIES_CONFIG_DIR value in  
+# bookkeeper/bookkeeper-server/bin/bookkeeper script file.
+# localBookiesConfigDirectory=/tmp/localbookies-config
```

Comment:
This is used only for testing and devs maybe it is better to move to the very bottom of the file?

Reply:
going to remove this config